research-app/src/App.vue

```diff
@@ -46,9 +67,14 @@
 <script lang="ts">
 import * as moment from "moment";
 import * as screenfull from "screenfull";
+import vSelect from 'vue-select';
+import Vue from 'vue';
+import 'vue-select/dist/vue-select.css';
 import { Component, Prop, Watch } from "vue-property-decorator";
 import { fmtDegLat, fmtDegLon, fmtHours } from "@wwtelescope/astro";
 
+Vue.component("v-select", vSelect);
```

Comment:
I've been putting these calls in the `main.ts` file. It would be good to keep them together there, if there's not a specific reason for doing things differently here.